template/src/ontology/Makefile.jinja2

```diff
@@ -749,60 +750,76 @@ all_components: $(OTHER_SRC)
 
 .PHONY: recreate-components
 recreate-components:
-	$(MAKE) COMP=true IMP=false MIR=true PAT=true IMP_LARGE=false all_components -B
+	$(MAKE) {% for component in project.components.products %}--assume-new=$(TMPDIR)/stamp-component-{{ component.filename }} \
```

Comment:
We should really make a `make` reference with "useful make tricks for ontology pipelines" with all the could things you have taught me in the last years. `--assume-new`? WOW.

Reply:
The reference exists, it’s called the GNU Make Manual. :stuck_out_tongue:  Available from `info make` on all decent operating systems, or [here](https://www.gnu.org/software/make/manual/html_node/index.html).